pagopa/io-app
# feat(Firma con IO): [SFEQS-1684] Add cancelled FCI request screen

### ⚠️ This PR depends on #4666

## Short description
This PR adds a new screen for signature requests in the `CANCELLED` status.

## List of changes proposed in this pull request
- locales/en/index.yml & locales/it/index.yml: Adds locale for the `CANCELLED` request screen.
- ts/features/fci/components/SuccessComponent.tsx: Adds screen for `CANCELLED` requests;
- ts/features/fci/components/__tests__/SuccessComponent.test.tsx: Adds test for the `CANCELLED` request screen;
- package.json: update `api_fci` version.

## How to test
Start `io-dev-api-server` with the following configuration by switching on the `SFEQS-1684-add-cancelled-fci-screen` branch and check if the screen gets rendered properly: 
```
...
fci: {
      canceledCount: 1,
      response: {
        getFciResponseCode: 200
      }
    },
...
```
